.gitignore

```diff
@@ -42,9 +42,12 @@ venv/
 .idea/
```

Comment:
Need to exclude everything under `generated` completely.

Reply:
removed in 09b5f9cd6b08f41f9226ab61decbf54bb237b99b